Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
```

Comment:
### [concurrent-ruby](http://www.concurrent-ruby.com)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.2.2 → 1.2.3

[_[change-log](https://github.com/ruby-concurrency/concurrent-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/ruby-concurrency/concurrent-ruby), [gem-diff](https://my.diffend.io/gems/concurrent-ruby/1.2.2/1.2.3)_]





<details>
<summary>Commits</summary>

A change of **25** commits. See the full changes on [the compare page](https://github.com/ruby-concurrency/concurrent-ruby/compare/v1.2.2...v1.2.3).

These are the first 10 commits:
- (7d58ec8) [Check the first Concurrent::Map default_proc argument is a C…](https://github.com/ruby-concurrency/concurrent-ruby/commit/7d58ec8998c226fd407fc7632b40bebdb631b8ca)
- (c587821) [Update comment about ::Hash on CRuby, it's not fully thread-…](https://github.com/ruby-concurrency/concurrent-ruby/commit/c587821ba5dd7e3fa0f758e0624ee4e47423f8f1)
- (e7ff945) [Update comment about ::Array on CRuby, it's not fully thread…](https://github.com/ruby-concurrency/concurrent-ruby/commit/e7ff945661a93a2e094b3259f23ccb1b3eea9bcf)
- (9f40827) [Fix TimerTask `:execution_interval` docs](https://github.com/ruby-concurrency/concurrent-ruby/commit/9f40827be9a8a192a6993a8d157bd3ed0662ada0)
- (483dd33) [Fix TimerTask docs to not refer to `#execute` as "blocking" …](https://github.com/ruby-concurrency/concurrent-ruby/commit/483dd33720f57799a913c9ae4419930630a39a1f)
- (3bfe39e) [Fix TimerTask example output (#1003)](https://github.com/ruby-concurrency/concurrent-ruby/commit/3bfe39e77958876403cefa3a2d89e86c7bdb5f9d)
- (bf966c0) [Fix broken CI due to rake-compiler error on Ruby < 2.6](https://github.com/ruby-concurrency/concurrent-ruby/commit/bf966c05654c490ec8f4db8e5ab2bf9b7e64a006)
- (1982b92) [Fix doc typo: yeild → yield](https://github.com/ruby-concurrency/concurrent-ruby/commit/1982b92daa8aee0d88db5212a61b790142c4106f)
- (42d2766) [Fix DaemonThreadFactory - reuse single Java thread factory](https://github.com/ruby-concurrency/concurrent-ruby/commit/42d2766023ffe5e95b62b46d29fe9a796af6a3fe)
- (dadc2ad) [Fix sporadic failures testing with JRuby (#1012)](https://github.com/ruby-concurrency/concurrent-ruby/commit/dadc2ad81f8c57a03400585187c3405c96713198)


</details>